### ISSUE TITLE
Active internal links.

### ISSUE THREAD

**Issue by shapkarin** [aeternity/superhero-ui#930]
I shared some internal superhero.com link inside the comment and it's not looks like usual link and user can't follow it. I think that it's a nice idea to make such links active.

Example comment can be found at: https://superhero.com/tip/2406_v1/

**Comment by davidyuk** [comment]
We can implement parsing and highlighting links in the comments in general, not just for superhero links

**Comment by shapkarin** [comment]
> We can implement parsing and highlighting links in the comments in general, not just for superhero links

I think we need to confirm with @maxkpower about external links in comments. But I'm pretty sure that active internal links will be safe and useful.

**Comment by maxkpower** [comment]
Absolutely! Great idea!

**Comment by shapkarin** [comment]
@maxkpower what fo you think about active external links in comments? I'm not sure about that.

**Comment by maxkpower** [comment]
I would say we make both active. Or are there any security / phishing concerns?

**Comment by shapkarin** [comment]
> I would say we make both active. Or are there any security / phishing concerns?

Not actually, but usually social networks use proxy to inform user that he goes out of website and that link can cause something unexpected by service.

**Comment by shapkarin** [comment]
Also they have some checklist and blacklists with bas domains to prevent redirect such links from their proxy.

**Comment by shapkarin** [comment]
@davidyuk what do you think about my messages above? I mean about some proxy that redirects to the outside links after user will confirm that redirection? Like VK, Linkedin and Facebook does it and other social networks?

**Comment by shapkarin** [comment]
@davidyuk what do you think about my messages above? I mean about some kind of proxy that redirects to the outside links after user will confirm that? Like VK, Linkedin and Facebook does it and other social networks? Anyway we will add a server soon, so we can use it for that propose as well.

**Comment by davidyuk** [comment]
Every tip already contains a link to an external service. Proposing to have a separate issue to discuss it. From my perspective to have an additional redirect confirmation screen is overkill now. Also, SSR is not needed for that (you can redirect the user to a SPA view).

**Comment by shapkarin** [comment]
@davidyuk agree, it's overkill, but only for now. I will open an issue about that feature later.

**Comment by shapkarin** [comment]
@mradkov @davidyuk how about to do that after SSR integration?

**Comment by davidyuk** [comment]
Let's see when we finish SSR integration
